MapleServer2/Types/StatDistribution.cs

```diff
@@ -9,8 +9,9 @@ namespace MapleServer2.Types
     public class StatDistribution
     {
         public int TotalStatPoints { get; private set; }
-        // TODO: implement Dictionary to keep track of points earned from quest, trophy, exploration, prestige
-        // naming convention: PointsFromQuest, PointsFromTrophy, PointsFromExploration, PointsFromPrestige
+        public Dictionary<byte, int> OtherStats { get; private set; }
+        // key: 01 PointsFromTrophy, 02 PointsFromQuest, 03 PointsFromExploration, 04 PointsFromPrestige
```

Comment:
Maybe if you turn this into an enum, you don't need this comment, then in lines 26-29, you can use the enum there directly, which also helps self-document the code